sympy/stats/frv_types.py

```diff
@@ -537,6 +624,11 @@ def Rademacher(name):
     >>> density(X).dict
     {-1: 1/2, 1: 1/2}
 
+    Returns
+    =======
+
+    A RandomSymbol.
```

Comment:
If you denote the parameter by type, you should use a single class, object name.
It may render up in HTML docs as bold text, so it may not look good.

Reply:
I have used it because all current docs of stats follows the same pattern of return type. Should I go for changing it to `RandomSymbol` everywhere in `stats`?

Reply:
```
Returns
=======

RandomSymbol
```
I used this but it didn't appear bold text in the docs locally.

Reply:
They should denote type, not some explanations.
If there should be an explanation, it should be in the indented line below.
https://numpydoc.readthedocs.io/en/latest/format.html#sections

There can be many, but gradually, I'd expect things to be changed to `RandomSymbol`

Reply:
Sure! Will change it.